barretenberg/cpp/src/barretenberg/plonk_honk_shared/arithmetization/mega_arithmetization.hpp

```diff
@@ -72,16 +81,42 @@ template <typename FF_> class MegaArith {
     struct ClientIvcBenchStructuredBlockSizes : public MegaTraceBlocks<uint32_t> {
         ClientIvcBenchStructuredBlockSizes()
         {
+            // 2^19
             this->ecc_op = 1 << 10;
             this->pub_inputs = 1 << 7;
+            this->busread = 1 << 7;
             this->arithmetic = 198000;
             this->delta_range = 90000;
             this->elliptic = 9000;
             this->aux = 136000;
-            this->lookup = 72000;
-            this->busread = 1 << 7;
             this->poseidon2_external = 2500;
             this->poseidon2_internal = 14000;
+            this->lookup = 72000;
+
+            // Additional structurings for testing
```

Comment:
Was using these 2^18 and 2^20 traces for testing. They're somewhat of a pain to generate bc there's a bit of tailoring involved so was hoping to just keep them around for now but can remove if you object